### DISCUSSION TITLE
black model in safari

### DISCUSSION THREAD

**Discussion by tom-deadline** [google/model-viewer#2368] [Q&A]
Hey guys, 
In safari from time to time, the model will turn black and there are no textures, could you help me?
![Screen Shot 2021-05-07 at 14 07 20](https://user-images.githubusercontent.com/44176198/117605267-64282600-b168-11eb-9019-e582666026e0.png)

I haven't noticed this problem before.

**Comment by tom-deadline** [comment]
Update: now the issue is in ar placement, too.

Device: Google Pixel 3
AR priority is set to webxr
![Screenshot_20210507-150806](https://user-images.githubusercontent.com/44176198/117639246-c3039480-b194-11eb-9a2c-c8352781badb.png)
![Screenshot_20210507-150914](https://user-images.githubusercontent.com/44176198/117639228-be3ee080-b194-11eb-8889-5a567c5d86c0.png)

**Comment by elalish** [comment]
First step is to open it in our editor and check for validation errors/warnings: https://modelviewer.dev/editor/ If the model is clean, I'd recommend you open an issue and fill out the template in it's entirety so I have enough info to help you. That must include a repro link, or at the very least a link where I can download the problem model.

**Reply by tom-deadline** [reply]
@elalish I've sent you invite to your repo

**Reply by elalish** [reply]
What do you want me to do with your repo?

**Reply by tom-deadline** [reply]
you said you need to see the code

**Reply by elalish** [reply]
I don't want to search your repo; please just paste a URL that reproduces the problem. I can see all the code I need in chrome dev tools.

**Reply by tom-deadline** [reply]
https://viewarly.com/product?slug=eddie-pump-90-c-2 

this is the url, problem with webxr was solved, but model is still black in safari from time to time.

**Reply by elalish** [reply]
Thanks! No repro for me (unsurprisingly), so I'm pretty sure this is a dupe of #1622, which we're trying our best to debug.

**Reply by tom-deadline** [reply]
is there any temporary workaround?

**Reply by elalish** [reply]
Not that I'm aware of. Reload the page? Safari doesn't even throw any warnings when this happens, which has made it pretty tough to detect or debug.

**Reply by elalish** [reply]
At least I haven't heard of it reproducing on iOS, which is good.
